### ISSUE TITLE
TypeConverter into Blob compilation error.

### ISSUE THREAD

**Issue by jan-pintr** [Raizlabs/DBFlow#316]
Getting compilation errors in ```MyModel$Adapter```
```
error: incompatible types: Blob cannot be converted to byte[]
```
while trying to use
```
@com.raizlabs.android.dbflow.annotation.TypeConverter
public class BlobableConverter extends TypeConverter<Blob, Blobable> {
	private static final String CHARSET_UTF_16 = "UTF-16";

	@Override
	public Blob getDBValue(Blobable blobable) {
		try {
			byte[] stringBytes = blobable.getString().getBytes(CHARSET_UTF_16);

			return new Blob(stringBytes);
		} catch (UnsupportedEncodingException e) {
			throw new IllegalStateException(e);
		}
	}

	@Override
	public Blobable getModelValue(Blob blob) {
		try {
			String string = new String(blob.getBlob(), CHARSET_UTF_16);

			return new Blobable(string);
		} catch (UnsupportedEncodingException e) {
			throw new IllegalStateException(e);
		}
	}
}
```
This is example model
```
@Table(databaseName = AppDb.NAME)
public class MyModel extends BaseModel {
	@Column
	@PrimaryKey(autoincrement = true)
	long id;

	@Column
	@NotNull
	Blobable blobable;

	MyModel(){
		// used by DBFlow
	}

	public MyModel(long id, String test) {
		this.id = id;
		this.blobable = new Blobable(test);
	}

	public String getString() {
		return blobable.getString();
	}
}
```

Is Blob type converter supported? It look like there is missing ```getBlob()``` in ```bindToStatement()``` and ```bindToContentValues()``` methods.

**Comment by agrosner** [comment]
Appears to be a bug. The fickleness of the processor will be addressed in `3.0`. as for now ill fix this in `2.3.0`

**Comment by agrosner** [comment]
wait hold on, why are you using a `BlobConverter`? I guess this sense is not supported (as of yet). Meaning `Blob` is the resolved value.

**Comment by agrosner** [comment]
will bake support for this into new processor for `3.0.0`

**Comment by jan-pintr** [comment]
> wait hold on, why are you using a BlobConverter?

I need one column which is stored ciphered so I used own object and implemented ciphering in 'TypeConverter'. For now I am conveting into String.

**Comment by agrosner** [comment]
Fixed in 3.0/develop branch :)

**Comment by agrosner** [comment]
Fixed in 3.0/develop branch :)
